python_modules/libraries/dagster-postgres/dagster_postgres/event_log/event_log.py

```diff
@@ -181,6 +180,8 @@ def store_event(self, event: EventLogEntry) -> None:
             )
             res = result.fetchone()
             result.close()
+
+            # LISTEN/NOTIFY no longer used for pg event watch - preserved here to support version skew
```

Comment:
ah I see - old dagit version reading from the websocket, new user code writing to the DB? fun, sad that we need to still do this for the forseeable future. Is it at all expensive if nothing is listening?

Reply:
from what ive come to understand from the docs the cost is ~ zero for notify with no listeners